appveyor.yml

```diff
@@ -0,0 +1,16 @@
+version: '{build}'
+
+environment:
+    PATH:           'C:\Ruby200\bin;%PATH%'
+    VERSION_SUFFIX: -netstd

```

Comment:
> I don't think we should ever set VERSION_SUFFIX in the YAML

Agreed.

> It should be a clicky-clicky config item only.

Sure. For now. I have a secret long-term goal of having us commit all the version changes like  we used to, but I know that's contentious and definitely not a concern for today, so I'm not even going to mention it.